src/common/components/addressForm/addressForm.tpl.html

```diff
@@ -113,9 +113,49 @@
         </div>
       </div>
     </div>
-  </div>
-  <div class="row">
-    <div class="col-sm-6">
+    <!-- TODO: Turn this into component structure when we have time -->
+    <div ng-if="$ctrl.compactAddress">
+      <div class="col-sm-3">
+        <div class="form-group is-required" ng-class="{'has-error': ($ctrl.parentForm.addressRegion | showErrors) || $ctrl.loadingRegionsError}">
+          <label translate>{{'STATE'}}</label>
```

Comment:
Can you update the label structure to match the non-compact address version for accessibility?